generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
@CongLi  i agree with you to some extent, no need to add different variants.
But @huangyum think it is better to distinguish this,  so we can mark polarion case (2M and 1G are different case)pass or fail according to the test result easily in the future.